app/robots.ts

```diff
@@ -6,7 +6,7 @@ const robots = (): MetadataRoute.Robots => {
     rules: {
       userAgent: '*',
       allow: '/',
-      disallow: ['global-components/', 'test/', 'test-items/'],
+      disallow: ['/global-components/', '/test/', '/test-items/'],
```

Comment:
Got a lighthouse warning that these directories in robot.ts need a leading slash